lib/metanorma/standoc/requirement.rb

```diff
@@ -0,0 +1,65 @@
+module Metanorma
+  module Standoc
+    class Requirement
+      def satisfied
+        raise NotImplementedError("abstract method")
+      end
+
+      def cmd
+        raise NotImplementedError("abstract method")
+      end
+    end
+
+    class LatexmlRequirement < Requirement
+      def initialize
+        @recommended_version = '0.8.4'
+        @minimal_version = '0.8.0'
+        version_output, = Open3.capture2e("latexml --VERSION")
+        @actual_version = version_output&.match(%r{\d+(.\d+)*})
+      end
+
+      def satisfied
+        version = @actual_version
+
+        sanity_check(version)
+
+        if Gem::Version.new(version) < Gem::Version.new(@recommended_version)
+          version = "unknown" if version.to_s.empty?
+          header_msg = "WARNING latexmlmath version #{version} below 0.8.4!"
```

Comment:
Instead of `0.8.4` you meant `#{@recommended_version}`, right?

Reply:
Done